src/server/game/Entities/Player/CollectionMgr.h

```diff
@@ -78,12 +80,25 @@ class TC_GAME_API CollectionMgr
     HeirloomContainer const& GetAccountHeirlooms() const { return _heirlooms; }
 
     // Account-wide mounts
+    void LoadMounts();
+    void LoadAccountMounts(PreparedQueryResult result);
+    void SaveAccountMounts(SQLTransaction& trans);
+    void MountSetFavorite(uint32 spellId, bool favorite);
+
+    bool UpdateAccountMounts(uint32 spellId, bool isFavourite /*= false*/);
+    bool AddMount(uint32 spellId, bool isFavourite /*= false*/, bool factionMount = false);
+    static void LoadMountDefinitions();
+ 
+    MountContainer const& GetAccountMounts() const { return _mounts; }
 
 private:
     WorldSession* _owner;
 
     ToyBoxContainer _toys;
     HeirloomContainer _heirlooms;
+    MountContainer _mounts;
+
+    static MountDefinitionMap _mountDefinitions;
```

Comment:
Avoid using non constant static variables in header files since this may cause issues with dynamic linking.

Since you are using `_mountDefinitions` only in your compilation unit you could move it directly to `CollectionMgr.cpp` without referencing it in your `CollectionMgr` class or you make it non-static because you are always working directly on the `CollectionMgr` anyway.